nuspecs/ServiceFabric/content/PackageRoot/Config/eventFlowConfig.json

```diff
@@ -1,43 +0,0 @@
-{
-  "inputs": [
```

Comment:
@karolz-ms Should we keep this file so the provider name can be added automatically?

Reply:
I removed this file, because the content-file is only added in projects which uses the packages.config. Most of the default ServiceFabric project templates uses .NET Core or ASP.NET Core, so they won't get the eventFlowConfig.json file added to their project. Now the behavior is the same in all kind of projects.

Reply:
I think that makes sense